io.github.gamingdoom.Datcord.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=Datcord
+Exec=/app/lib/datcord/launch-datcord
```

Comment:
this should NO be an absolute path.

Reply:
What do you recommend to change it to? Will /app/bin work?